docs/user-guide/ingress.md

```diff
@@ -234,7 +265,7 @@ You can achieve the same by invoking `kubectl replace -f` on a modified Ingress
 
 ## Future Work
 
-* Various modes of HTTPS/TLS support (edge termination, sni etc)
+* Various modes of HTTPS/TLS support (eg: SNI, re-encryption)
```

Comment:
Are the future plans for SNI documented somewhere? If so, where can I find them? Also, same question about re-encryption?